### DISCUSSION TITLE
Github Copilot - Extension Activation Failed: "Class constructor cn cannot be invoked without 'new'"

### DISCUSSION THREAD

**Discussion by JPENEG** [community/community#43903] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I am getting the following error after signing up to copilot and installing the vscode extension:

`[ERROR] [default] [2023-01-10T20:54:13.088Z] GitHub Copilot could not connect to server. Extension activation failed: "Class constructor cn cannot be invoked without 'new'"`

I have tried logging out, restarting vs code and a shallow uninstall.

Help would be much appreciated.

**Comment by iamcjwilliams** [comment]
@JPENEG Sorry for the trouble here, Could you provide a few more details on how this can be replicated?

**Reply by JPENEG** [reply]
Sure thing @iamcjwilliams what would be helpful for me to provide?

Apple MacBook Pro M1Pro 14-inch, 2021 | 16gb Memory.

VS Code: Version: `1.74.3`

**Reply by iamcjwilliams** [reply]
Are you receiving this error at login or after initiating a function?

**Reply by JPENEG** [reply]
At login.

**Reply by Mirakud** [reply]
same error, i logout and re login but nothing work, i'm on Mac mini M1 Ventura 13.0.1, Version of vscode : 1.74.3 (Universal)

**Reply by nico-ns** [reply]
Same error on vscode version 1.74.3 on Windows 10

**Comment by Mirakud** [comment]
same error, i logout and re login but nothing work, i'm on Mac mini M1 Ventura 13.0.1, Version of vscode : 1.74.3 (Universal)

**Comment by Sidu28** [comment]
Also having this issue, same computer specs Apple MacBook Pro M1Pro 14-inch, 2021 | 16gb Memory.

**Comment by iamcjwilliams** [comment]
@JPENEG I have notified our copilot team of the issue

**Reply by Mirakud** [reply]
Thanks ^^

**Reply by Mirakud** [reply]
So ?

**Comment by FranBellCast** [comment]
Same issue here on a Macbook with M1

**Comment by Mirakud** [comment]
Any update ?
